### PR TITLE
temporary fix core tests

### PR DESCRIPTION
**Pull request by pinzart90**
Brings the testing time down to 15 mins:
image

### CODE REVIEW

File: src/Config/CS_SDK.props [hunk 1]
```diff
@@ -63,6 +63,12 @@
     <UseWPF>true</UseWPF>
     <UseWindowsForms>true</UseWindowsForms>
   </PropertyGroup>
+  <!--Temporary configuration to make core test projects load UI packages successfully -->
+  <PropertyGroup Condition=" '$(TestProjectDefaults)' == 'true' AND $(Platform.Contains('Windows')) ">
+    <TargetFramework>net6.0-windows</TargetFramework>
+    <UseWPF>true</UseWPF>
+    <UseWindowsForms>true</UseWindowsForms>
+  </PropertyGroup>
```
Comment (pinzart90) [inline src/Config/CS_SDK.props:71]: dynamoCoreTests and some other projects still want to load/test UI packages.
We need to separate them out, but for now we can consider them all UI projects

File: test/DynamoCoreTests/DynamoCoreTests.csproj [hunk 2]
```diff
@@ -52,12 +52,13 @@
...
-        <PackageReference Include="Moq" Version="4.18.4" GeneratePathProperty="true" />
-        <PackageReference Include="Castle.Core" Version=" 5.1.1" GeneratePathProperty="true" />
-        <PackageReference Include="System.Threading.Tasks.Extensions" Version="4.5.4" GeneratePathProperty="true" Condition=" '$(TargetFramework)' == 'net48' " />
+    <!--moq and deps - copied to test_dependencies folder-->
+    <PackageReference Include="Moq" Version="4.18.4" GeneratePathProperty="true" />
+    <PackageReference Include="Castle.Core" Version=" 5.1.1" GeneratePathProperty="true" />
+    <PackageReference Include="System.Threading.Tasks.Extensions" Version="4.5.4" GeneratePathProperty="true" Condition=" '$(TargetFramework)' == 'net48' " />
 
-        <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
+    <PackageReference Include="JUnitTestLogger" Version="1.1.0" />
```
Comment (pinzart90) [inline test/DynamoCoreTests/DynamoCoreTests.csproj:60]: New package reference for tests - JUnitTestLogger
This logger does not log std output and std err in the reports

**Comment by pinzart90** [comment]
Passed here https://master-5.jenkins.autodesk.com/job/Dynamo/job/DynamoSelfServe/job/pullRequestValidation/10625/

File: test/DynamoCoreTests/DynamoCoreTests.csproj (hunk 2 above)
Comment (aparajit-pratap) [reply test/DynamoCoreTests/DynamoCoreTests.csproj:60]: How is this used?

File: src/Config/CS_SDK.props (hunk 1 above)
Comment (aparajit-pratap) [reply src/Config/CS_SDK.props:71]: Can you explain how this change brings down the testing time?

Comment (pinzart90) [reply src/Config/CS_SDK.props:71]: Less logs and less exceptions thrown during tests

File: test/DynamoCoreTests/DynamoCoreTests.csproj (hunk 2 above)
Comment (pinzart90) [reply test/DynamoCoreTests/DynamoCoreTests.csproj:60]: Dotnet test command loads the logger dll from the app output path. Based on the name specified logger: junit

File: src/Config/CS_SDK.props (hunk 1 above)
Comment (aparajit-pratap) [reply src/Config/CS_SDK.props:71]: How was it working without setting `UseWpf` and `UseWinforms` to `true` before?

Comment (aparajit-pratap) [reply src/Config/CS_SDK.props:71]: What evidence suggests this: `dynamoCoreTests and some other projects still want to load/test UI packages.`, modules loaded when running dynamocoretests?

Comment (pinzart90) [reply src/Config/CS_SDK.props:71]: There might be junk packages (with you refs) that are leftover from other UI tests

Comment (pinzart90) [reply src/Config/CS_SDK.props:71]: Well, the evidence was in the logs/exceptions. Ex “cannot find windowsBase”

**Review by aparajit-pratap — APPROVED** [review]

**Pull request merged**